Dockerfile

```diff
@@ -21,11 +21,11 @@ COPY ${REMOTE_SOURCE}/bundle bundle
 COPY ${REMOTE_SOURCE}/version version
 COPY ${REMOTE_SOURCE}/.bingo ./.bingo
 ADD ${REMOTE_SOURCE}/Makefile ${REMOTE_SOURCE}/main.go ${REMOTE_SOURCE}/go.mod ${REMOTE_SOURCE}/go.sum ./
-
+USER 0
```

Comment:
The new image seems to default to having a user `default` with the working directory in `/opt/app-root/src`. As we do not need the "GOPATH-style" of working directory, does it make sense to update the Dockerfile to use the defaults instead?

Reply:
Maybe.  but as @periklis noted this is generated from Dockerfile.in.  Additional, `REMOTE_SOURCE` is deprecated which means there are additional changes to be made.  Adding the user was the same change I made to the CLO which did not seem to have any adverse affects.

Reply:
I realized too late that I made the comments on the generated file and not the source :slightly_smiling_face: